dbms/src/Functions/FunctionsString.cpp

```diff
@@ -4463,6 +4465,13 @@ class PadUTF8Impl : public IFunction
     }
 };
 
+static Int64 getResult(const ColumnString::Chars_t & chars, size_t size, size_t offset)
```

Comment:
maybe we should move this function inside `FunctionASCII`

Reply:
> maybe we should move this function inside `FunctionASCII`

done